src/components/Icons/HomeIcon.js

```diff
@@ -0,0 +1,15 @@
+import { FaHome } from "react-icons/fa";
+import { IconLink } from './styled';
+
+const HomeButton = () => {
+  return (
+    <IconLink
+      title="Home"
```

Comment:
href="#/"
We can go ahead and this attribute and location.

Reply:
I made a mistake. I am sorry but can you update this another time please too
href="/"
Per discord convo with igni the # shouldn't be included. It actually was never included. That was a mistake I made when I copied my url to what I thought it should be.
Thank you!

Reply:
This is perfect! :+1: